brjs-sdk/sdk/libs/javascript/br-i18n/src/br/i18n/Translator.js

```diff
@@ -386,6 +386,10 @@ Translator.prototype._getTranslationForKeyOrUndefined = function(token) {
 	token = token.toLowerCase();
 
 	if (!this.tokenExists(token)) {
+		if (typeof window.console === "undefined") {
```

Comment:
It would probably be better to conditionally log to the console rather than replacing the console. e.g.
```
var logConsole = (window.jstestdriver) ? jstestdriver.console : window.console;
if (logConsole) {
logConsole.warn('Unable to find a replacement for the i18n key "' + token + '"');
}
```